teps/0091-trusted-resources.md

```diff
@@ -364,26 +373,70 @@ type KeylessRef struct {
 
 ### Configuration
 
-Reconciler will respond to verification failure based on ConfigMap settings to:
+Reconciler will respond to verification failure based on ConfigMap and Verification Policy exsitence settings to:
 1) Return error in reconciler, mark the status as error with `FailedVerification` reason;
-2) Not return error, but mark the TaskRun/PipelineRun as `FailedVerification`;
+2) Not return error, but warn the log and update the status condition with `FailedVerification` reason;
 3) Skip the verification;
 
-Configuration sample:
-```yaml
+ ```yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: feature-flag
   namespace: tekton-pipelines
 data:
-  verification-policy: enforce
+  no-match-policy: "deny"
 ```
 
-`verification-policy`. (Optional, `enforce`, `warn` or `skip`, default `skip`):
- * `enforce`: Failing verification will mark the taskruns/pipelineruns as failed.
- * `warn`: Log warning but don't fail the taskruns/pipelineruns.
- * `skip`: Directly skip the verification.
+`no-match-policy`. (Optional, `allow`, `deny` or `warn`, default `allow`):
+ * `deny`: Reject the resource and fail the taskrun/pipelinerun if no matching policies are found.
+ * `allow`: Allow the resource if no matching policies are found.
```

Comment:
Since warn is a no-op as far as execution goes, do we need allow? Cant it be just deny or warn?

Reply:
I think we need? This is for the feature flag not the policy mode